src/content/docs/en/recipes/getImage.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Build a custom Picture Component 
+description: Learn how to build a custom Picture Component using the getImage function
+i18nReady: true
+type: recipe
+---
+
+The `getImage()` function is used for generating images that will be used elsewhere instead of directly in the HTML. In this recipe, you will learn how to create your own `<Picture>` component using the `getImage()` function.
+
+
+## Prerequisites
+- A project with [SSR](/en/guides/server-side-rendering/) (`output: 'server'`) enabled
+
+
+## Recipe
+
+1. Create a new Astro component and import the `getImage()` function
+
+    ```astro title="src/components/MyCustomPictureComponent.astro"
+    ---
+     import { getImage } from "astro:assets";
+    ---
+
+    ```
+
+2. Destructure the `props` from `Astro.props`. You can also use TypeScript to type the props. 
+
+   ```astro title="src/components/MyCustomPictureComponent.astro"
+    ---
+    interface Props {
+        mobileImgUrl: string | ImageMetadata;
+        desktopImgUrl: string | ImageMetadata;
+        alt: string;
+    }
+
+    import type { ImageMetadata } from "astro";
+    const { mobileImgUrl, desktopImgUrl, alt } = Astro.props;
+    import { getImage } from "astro:assets";
```

Comment:
You should move the imports at the top of the file, just after the opening `---`